### DISCUSSION TITLE
Text (Excel table paste) to SQL Insert statement

### DISCUSSION THREAD

**Discussion by amlloyd** [CorentinTh/it-tools#1341] [Ideas]
**WHAT** 
I think a great addition would be a SQL insert statement generator. I envision pasting a table of data from Excel or elsewhere into a text box, and it will convert into standard SQL INSERT statement. 

**USE CASE**
I am often have users sending me data in excel or similar that needs to be manually inserted into various SQL tables. Would be super handy to be able to quickly generate an INSERT statement that I can paste into my SQL IDE and execute.

**Comment by sharevb** [comment]
Hi @amlloyd , implemented here : https://sharevb-it-tools.vercel.app/excel-to-data and https://sharevb-it-tools.vercel.app/csv-to-data

And if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
